src/app.js

```diff
@@ -1 +1,57 @@
 'use strict';
+
+const randomDigits = require('./randomNumbers');
+const countBullsAndCows = require('./getCountBullAndCows');
+const readline = require('readline');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function promptUser(prompt) {
+  return new Promise((resolve) => {
+    rl.question(prompt, (answer) => {
+      resolve(answer);
+    });
+  });
+}
+
+function printResult(result) {
+  // eslint-disable-next-line no-console
+  console.log(`Bulls: ${result.bulls}, Cows: ${result.cows}`);
+};
+
+function printMessage(message) {
+  // eslint-disable-next-line no-console
+  console.log(message);
+};
+
+async function playGame() {
+  const numbers = randomDigits();
+  let answear;
+
+  if (!answear) {
+    printMessage('Welcome to Bulls and Cows game!');
+  }
+
+  while (answear !== numbers) {
```

Comment:
It's better to use a different comparison operator like '!==', use 'while (answear.toString() !== numbers.toString())'